package.json

```
@@ -25,14 +25,14 @@
     "@opam/dune": "*",
```

```suggestion
    "@opam/dune": ">=1.11",
```
because you have updated dune-project.